utils/container-runtime/src/runtime.rs

```diff
@@ -316,7 +316,7 @@ impl ContainerRuntime {
         }
     }
 
-    #[tracing::instrument(level = "debug", skip_all, fields(container_name))]
+    #[tracing::instrument(level = "debug", skip_all, fields(%container_name))]
```

Comment:
No idea why, but without `%` these values are not showing up in traces ... this is a really unexpected behavior and I will look into it some day.